2-mandatory/3-stocks.js

```diff
@@ -48,9 +56,18 @@ function getAveragePrices(closingPricesForAllStocks) {
     The price change value should be rounded to 2 decimal places, and should be a number (not a string)
 */
 function getPriceChanges(closingPricesForAllStocks) {
-    // TODO
+      let result = [];
+
+    for(let i = 0; i < closingPricesForAllStocks.length; i++){
+        let diff = 0;
+        diff = closingPricesForAllStocks[i][4] - closingPricesForAllStocks[i][0];
+        result.push(parseFloat(diff.toFixed(2)));
+    }
+
+    return result;
 }
 
```

Comment:
Better solution 

function getPriceChanges(closingPricesForAllStocks) {
    let changes = [];

    for(let pricesForStock of closingPricesForAllStocks) {
        changes.push(getPriceChangeForStock(pricesForStock))
    }

    return changes;
}

function getPriceChangeForStock(pricesForStock) {
    let priceChange = pricesForStock[pricesForStock.length - 1] - pricesForStock[0]
    return roundTo2Decimals(priceChange);
}